.github/workflows/commit-trailers.yml

```
@@ -35,7 +39,17 @@ jobs:
         run: node --test scripts/check-commit-trailers.test.mjs
 
-      - name: Validate attribution trailers
+      - name: Validate attribution trailers (branch commits)
         run: >
           node scripts/check-commit-trailers.mjs
           --range "${{ github.event.pull_request.base.sha }}..${{ github.event.pull_request.head.sha }}"
+
+      - name: Validate PR-body trailer (post-squash attribution lands here)
+        env:
+          PR_BODY: ${{ github.event.pull_request.body }}
+        run: |
```

actionlint (1.7.12)

[error] unexpected end of input while parsing variable access, function call, null, bool, int, float or string. expecting "IDENT", "(", "INTEGER", "FLOAT", "STRING"

(expression)